src/components/FormFields.tsx

```diff
@@ -0,0 +1,34 @@
+type Props = {
+  name: string,
+  min: number,
+  max: number,
+  step: number,
+  defaultValue: number,
+  onChange: (value: number) => void,
+};
+
+export const Input: React.FC<Props> = ({
+  name,
+  min,
+  max,
+  step,
+  defaultValue,
+  onChange,
+}) => {
+  return (
+    <label htmlFor={name}>
+      {`${name}`}
+      <input
+        id={name}
+        type="range"
+        defaultValue={defaultValue}
+        step={step}
+        min={min}
+        max={max}
+        onChange={(event) => {
+          onChange(+event.target.value);
+        }}
```

Comment:
it is bad practice to make anonymous arrow functions in jsx.
see comment above how to change handler
```suggestion
        onChange={onChange}
```